src/ng/services/table.js

```diff
@@ -1,3 +1,58 @@
+class Dom {
+	constructor(element) {
+		this.element = element;
+	}
+
+	static get empty() {
+		return new Dom(null);
```

Comment:
The better idea will be the base class with empty methods , check on nulls is always worse